ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -303,6 +303,8 @@
 {% include 'eos/management-tech-support.j2' %}
 {# platform apply #}
 {% include 'eos/platform-apply.j2' %}
+{# stun #}
+{% include 'eos/stun.j2' %}
```

Comment:
we try to apply the config in the right order so that it is "easier" to compare with a show run

on ceos stun is before `management-tech-support`and dot1x so let's move it up above this (the management stuff above it not really in the right order comparing to show run for now)

```
stun
   client
      refresh interval 42 seconds
      !
      server-profile TEST
         ip address 42.42.42.42
      !
      server-profile TEST2
         ip address 66.66.66.66
   !
   server
      local-interface Ethernet42
!
dot1x system-auth-control
!
management tech-support
   policy show tech-support
      include command "show toto"
!
end
```